### PR TITLE
FLAG-49 Improve loading speed of Patient Dashboard

### PR DESCRIPTION
**Pull request by slubwama**
https://issues.openmrs.org/browse/FLAG-49

### CODE REVIEW

**Comment by HerbertYiga** [comment]
Did you see the conflicting files?

**Comment by slubwama** [comment]
@ yes but they are as a result of a merge that was not meant to be. https://github.com/openmrs/openmrs-module-patientflags/pull/25
Not sure it that had passed all criteria.

**Comment by slubwama** [comment]
@ssmusoke kindly look into this

**Comment by slubwama** [comment]
@HerbertYiga Conflicts have been resolved CC @ssmusoke

**Comment by HerbertYiga** [comment]
could you check the failing travis error logs!!

**Comment by slubwama** [comment]
> could you check the failing travis error logs!!

Expected feature release number in range of 9 to 13, but got: 8
The command "~/bin/install-jdk.sh --target "/home/travis/oraclejdk8" --workspace "/home/travis/.cache/install-jdk" --feature "8" --license "BCL"" failed and exited with 3 during .

**Comment by dkayiwa** [comment]
@slubwama can you switch to openjdk as done here? https://github.com/openmrs/openmrs-module-webservices.rest/pull/386/files

**Comment by slubwama** [comment]
@dkayiwa  thanks

**Comment by ssmusoke** [comment]
@slubwama 

1. Add a loading indicator - I think you can use the FontAwesome spinner 
2. Add a message when there are no flags match the patient - you can use the text coreapps.none (thats what some of the widgets show)

**Comment by slubwama** [comment]
@ssmusoke  Done with Spinner

File: omod/src/main/webapp/fragments/patientflagsDashboard.gsp [hunk 1]
```diff
@@ -1,15 +1,26 @@
...
     </div>
 
     <div class="info-body">
-    	<ul>
-            <% flags.each { flag -> %>
-                <li><span style=" ${ flag.priority.style } "> ${ flag.evalMessage(config.patientId) } </span></li>
-            <% } %>
-    	</ul>
+        <script>
+            jq.post('${ ui.actionLink("processPatientFlags") }', {
```
Comment (ssmusoke) [inline omod/src/main/webapp/fragments/patientflagsDashboard.gsp]: @slubwama Why POST not GET since no data is being changed?

**Pull request merged**